tests/test_websocket.py

```diff
@@ -310,42 +311,56 @@ class WebsocketWriterTests(unittest.TestCase):
 
     def setUp(self):
         self.transport = unittest.mock.Mock()
-        self.writer = websocket.WebSocketWriter(self.transport)
```

Comment:
Why not to leave here `websocket.WebSocketWriter(self.transport, mask=False)` and just create own writer within `test_send_text_masked`? This won't require to touch every test.